### PR TITLE
Updates landing page for 8.6

### PR DESCRIPTION
**Pull request by KOTungseth**
## Summary

Addresses #242 to update the Welcome to Elastic Docs page and make it more visually appealing. 

[Preview](https://docs_2559.docs-preview.app.elstc.co/guide/index.html)

Welcome to Elastic Docs

## Ask

I want to make sure we're all aligned on the changes, so please leave any feedback, make updates, etc. 

## Plan

Let's wrap this up in time to merge for 8.6.0.

### CODE REVIEW

**Review by kilfoyle — APPROVED** [review]
Very nice! 👍 
I love having those cards for the GS guides. They really stand out now.

**Review by chriscressman — APPROVED** [review]
Looks good! :rocket:

File: extra/docs_landing.html [hunk 1]
```diff
@@ -1,238 +1,366 @@
...
-      <path d="M789.293 485.293C788.902 485.683 788.902 486.317 789.293 486.707L795.657 493.071C796.047 493.462 796.681 493.462 797.071 493.071C797.462 492.681 797.462 492.047 797.071 491.657L791.414 486L797.071 480.343C797.462 479.953 797.462 479.319 797.071 478.929C796.681 478.538 796.047 478.538 795.657 478.929L789.293 485.293ZM790 487L847 487L847 485L790 485L790 487ZM847 487L904 487L904 485L847 485L847 487Z" fill="#BD271E"/>
-      <path d="M993.707 419.293C993.317 418.902 992.683 418.902 992.293 419.293L985.929 425.657C985.538 426.047 985.538 426.681 985.929 427.071C986.319 427.462 986.953 427.462 987.343 427.071L993 421.414L998.657 427.071C999.047 427.462 999.681 427.462 1000.07 427.071C1000.46 426.681 1000.46 426.047 1000.07 425.657L993.707 419.293ZM992 420V440H994V420H992Z" fill="#BD271E"/>
-      <text fill="black" xml:space="preserve" style="white-space: pre" font-family="Inter" font-size="26" font-weight="bold" letter-spacing="0em"><tspan x="9" y="24.9545">Components of the Elastic Stack</tspan></text>
-      </svg>
-      
+<div class="row my-4">
```
Comment (bmorelli25) [inline extra/docs_landing.html:52]: I think overall this looks great, but I'd love to see some of this whitespace removed from this class.
Screen Shot 2022-11-16 at 11 55 18 AM

File: extra/docs_landing.html [hunk 2]
```diff
@@ -1,238 +1,366 @@
...
+      <a class="inline-block mr-3" href="https://www.elastic.co/videos">Videos & Webinars</a>
+   </p>
+   </div>
+   <div class="col-md-6 col-12">
+     <iframe class="vidyard_iframe" src="https://play.vidyard.com/qAS1W24928bBrc2sv5f5Ti?v=4.3&amp; width="640" height="360" scrolling="no" frameborder="0" allowfullscreen="" /></iframe>
+   </div>
+  </div>
 
-  <h2>Get started</h2>
+  <h3>Get started</h3>
```
Comment (bmorelli25) [inline extra/docs_landing.html:218]: Similarly, I don't think this margin is needed.
Screen Shot 2022-11-16 at 11 57 41 AM

File: extra/docs_landing.html [hunk 3]
```diff
@@ -1,238 +1,366 @@
...
         </div>
-      </td>
-    </tr>
-  </table>
-</div>
+      </a>
+    </div>
+  </div>
+
+  <div class="my-5">
```
Comment (bmorelli25) [inline extra/docs_landing.html:282]: I think reducing the margin around the my-5 class also makes sense.

Screen Shot 2022-11-16 at 11 59 22 AM

**Review by florent-leborgne — APPROVED** [review]
This is such a great improvement! 

- I like Brandon's suggestion to better use the space
- Maybe we can strip out the "Learn how to" part in the cards to gain a few words?

**Review by bmorelli25 — APPROVED** [review]
Much less wasted space! Looks great.

**Pull request merged**